fjord/redirector/__init__.py

```diff
@@ -17,4 +17,4 @@ def get_redirectors():
     return list(_REDIRECTORS)
 
 
-from .base import build_redirect_url, Redirector
+from .base import build_redirect_url, Redirector, RedirectParseError
```

Comment:
Is this a "import for export" type of thing? I don't see it in the `__all__` definition above.